arabic_vocab

```diff
@@ -0,0 +1 @@
+VOCABS['arabic'] = 'ءآأؤإئابةتثجحخدذرزسشصضطظعغـ‌‍‎‏فقكلمنهوىيًٌٍَُِّْٯٰپچژڜڢڤڥڧڨکگی' + '٠١٢٣٤٥٦٧٨٩'
```

Comment:
Would you mind removing this file please? :pray:

Reply:
I have deleted it. Hope it is all set now. Thanks @fg-mindee for your follow-up and support.